### DISCUSSION TITLE
Hoymiles HMS Series mit WLAN

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#1031] [General]
Interessant... jetzt gibt es auch HMS Wechselrichter ohne CMT2300A Chip 😞 

![Hoymiles HMS WLAN Versions](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/a2cb50b4-ce15-4abc-a13c-41a1a09e1371)

**Comment by Sorbat** [comment]
Was meinst du? 
Die gibt es doch schon seit Ewigkeiten. In der Facebook hoymiles Gruppe ist das das meiste gepostete Problem, dass jemand den WLAN WR nicht mit der DTU verbunden bekommt oder die CMT Version nicht mit dem WLAN 😅

**Reply by Manos1966** [reply]
Auch die HMS Serie? 🤦‍♂️

**Reply by Sorbat** [reply]
Ja klar. Welche denn noch?

**Reply by Manos1966** [reply]
Hmmm, das bedeutet es gibt weder NRF24L01+ noch CMT2300A-fähige Geräte auf dem Neu-Markt ? 🙈

**Reply by Sorbat** [reply]
~~Ja, soweit ich weiß ist das richtig.~~
Die HM- mit nrf sind ja ohnehin nicht mehr gut zu bekommen. 
~~Die HMS Modelle mit cmt scheinen wohl auch langsam auszulaufen, da sie immer schlechter verfügbar werden, wobei ich gesehen hatte, dass da teils noch ordentliche Lagerbestände sind (in Polen zb.)~~
Und das aktuelle Modell HMS-xxxW-.. hat nur noch wlan 😒

**Reply by schlimmchen** [reply]
> Die HMS Modelle mit cmt laufen gerade auch aus

Gibt's da auch eine verlässliche Quelle für diese Aussage?

**Reply by Sorbat** [reply]
Nein, habe ich nicht direkt.
Die Aussage bezog sich auf meine Erfahrungen der Verfügbarkeit. 
Wenn man die Verfügbarkeit der HMS mit jener von vor einigen Monaten vergleicht, ist es deutlich enger geworden, und die wlan Modelle sind hier deutlich präsenter.
Ich habe meine Aussage in der letzten Nachricht etwas umformuliert

**Reply by Sorbat** [reply]
@schlimmchen 
Hatte Rückmeldung von hoymiles bekommen. Aktueller Stand ist, dass es keine offizielle Info darüber gibt, dass hms cmt Modelle jetzt ausgelistet werden.

![hm](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/72571258/ad32da5b-66c4-41e5-ab38-a8e712eae4f4)
Stand: 07.06.2024

Vielleicht hängt dann die schlechtere Verfügbarkeit der Modelle einfach mit einer geringeren Nachfrage ab, auch im Vergleich zu den wlan Modellen

**Reply by schlimmchen** [reply]
Danke, dass du extra nachgehakt hast.

Ich sehe die "schlechte" Verfügbarkeit eher in *erhöhter* Nachfrage begründet...
